examples/grpc/security-outbound/pom.xml

```diff
@@ -0,0 +1,101 @@
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.examples.grpc</groupId>
+        <artifactId>helidon-examples-grpc-project</artifactId>
+        <version>1.0.3-SNAPSHOT</version>
+    </parent>
+    <artifactId>helidon-examples-grpc-security-outbound</artifactId>
+    <name>Helidon gRPC Server Examples Outbound Security</name>
+
+    <description>
+        Examples of outbound security when using gRPC services
+    </description>
+
+    <properties>
+        <mainClass>io.helidon.grpc.examples.security.outbound.SecureServer</mainClass>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.examples.grpc</groupId>
+            <artifactId>helidon-examples-grpc-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.helidon.grpc</groupId>
+            <artifactId>helidon-grpc-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.grpc</groupId>
+            <artifactId>helidon-grpc-server</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.security.integration</groupId>
+            <artifactId>helidon-security-integration-grpc</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.bundles</groupId>
+            <artifactId>helidon-bundles-config</artifactId>
+           <version>${project.version}</version>
+       </dependency>
+        <dependency>
+            <groupId>io.helidon.bundles</groupId>
+            <artifactId>helidon-bundles-webserver</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.security.integration</groupId>
+            <artifactId>helidon-security-integration-jersey</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+           <groupId>io.helidon.bundles</groupId>
+           <artifactId>helidon-bundles-security</artifactId>
+           <version>${project.version}</version>
+           <exclusions>
+               <exclusion>
+                   <groupId>com.google.code.findbugs</groupId>
```

Comment:
Why the exclusion ? This should rather be excluded where it's coming from, or marked optional if this is a helidon module.

Reply:
This came from copying things from `examples/security/attribute-based-access-control/pom.xml` which has the same exclusion. I'm happy to remove it though.

Reply:
@tomas-langer Can you comment on that ?

Reply:
This is a left-over from the time Google login was part of the security bundle (if I remember correctly).
They had a runtime dependency on this library, which I do not like, so I excluded it.